src/renderers/shared/fiber/ReactFiberCompleteWork.js

```diff
@@ -120,13 +122,24 @@ module.exports = function<T, P, I>(config : HostConfig<T, P, I>) {
       case HostContainer:
         return null;
       case HostComponent:
+        console.log('/host component', workInProgress.type);
         transferOutput(workInProgress.child, workInProgress);
-        if (workInProgress.alternate) {
+        const children = workInProgress.output;
+        const newProps = workInProgress.memoizedProps;
+        if (workInProgress.alternate && workInProgress.stateNode != null) {
```

Comment:
No, this is just to please Flow. We do have two fields to play with though. Could just use output potentially.